storage/mroonga/vendor/groonga/plugins/tokenizers/mecab.c

```diff
@@ -241,7 +241,7 @@ chunked_tokenize_utf8(grn_ctx *ctx,
       last_delimiter = current;
     }
 
-    if ((current - chunk_start) >= grn_mecab_chunk_size_threshold) {
+    if ((size_t)(current - chunk_start) >= grn_mecab_chunk_size_threshold) {
```

Comment:
I just remembered about [`ptrdiff_t`](https://en.cppreference.com/w/c/types/ptrdiff_t.html)… Have you considered it?

Reply:
Good idea. tried and got:
```
/source/storage/mroonga/vendor/groonga/plugins/tokenizers/mecab.c:244:44: error: comparison of integers of different signs: 'ptrdiff_t' (aka 'long') and 'size_t' (aka 'unsigned long') [-Werror,-Wsign-compare]
  244 |     if ((ptrdiff_t)(current - chunk_start) >= grn_mecab_chunk_size_threshold) {
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
implemented